radar_client_rb.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new "radar_client_rb", Radar::Client::VERSION do |gem|
   gem.summary = gem.description = "Read/Write Radar Resources from Redis through Ruby"
   gem.files = Dir.glob("lib/**/*")
 
-  gem.required_ruby_version = ["~> 2.6", ">= 2.6"]
+  gem.required_ruby_version = [">= 2.6"]
```

Comment:
Shouldn’t this be `>= 2.7`?

Reply:
Did we do anything to make it _not_ compatible with 2.6? If not why change this?

Reply:
I have added Ruby 2.6 back to the test matrix.